sunpy/spectra/spectrogram.py

```diff
@@ -1144,7 +1144,7 @@ def intersect_time(specs):
         # XXX: Could do without resampling by using
         # sp.t_init below, not sure if good idea.
         specs = [sp.resample_time(delt) for sp in specs]
-        cut = [sp[:, (start - sp.t_init) / delt:] for sp in specs]
+        cut = [sp[:, int((start - sp.t_init) / delt):] for sp in specs]
```

Comment:
this file is complicated... I won't look too much in detail, but if this makes travis happy, I'm happy.